src/hooks/scheduledEvent/useScheduledEvents.ts

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect, useCallback } from 'react';
+
+import { ScheduledEvent, ScheduledEventFilter } from 'generated/sdk';
+import { useUnauthorizedApi } from 'hooks/common/useDataApi';
+
+export default function useScheduledEvents(filter?: ScheduledEventFilter) {
+  const [loading, setLoading] = useState(true);
+  const [scheduledEvents, setScheduledEvents] = useState<
+    Pick<
+      ScheduledEvent,
+      'id' | 'bookingType' | 'startsAt' | 'endsAt' | 'description'
+    >[]
+  >([]);
+
+  const unauthorizedApi = useUnauthorizedApi();
+
+  const refresh = useCallback(() => {
+    setLoading(true);
+    unauthorizedApi()
```

Comment:
I presume we will use authorizedApi latter :)

Reply:
I hope so :D